src/firefly/java/edu/caltech/ipac/firefly/server/query/AsyncTapQuery.java

```diff
@@ -71,14 +78,29 @@ public AsyncTapJob(String baseJobUrl) {
             this.baseJobUrl = baseJobUrl;
```

Comment:
could you please look at line 57, it is like 
HttpServices.postData(asyncTap.baseJobUrl + "/phase", new HttpServiceInput().setParam("PHASE", "RUN"), null);

please check if  "RUN" should be "EXECUTING"? or not?

Reply:
While "RUN" is not an execution phase, it should be used to "commit the job", if its phase is "PENDING".

According section 5.2 of [TAP 1.0](http://www.ivoa.net/documents/TAP/20100327/REC-TAP-1.0.html) (section 5.1.3 of [TAP 1.1](http://www.ivoa.net/documents/TAP/20181024/PR-TAP-1.1-20181024.html#tth_sEc5.1.3)):

The client runs the job by posting to the phase URL:
```
HTTP POST http://example.com/tap /async/42/phase
PHASE=RUN
```

As for the content of Table Info dialog, it's just the beginning, there will be much more there. The design of this dialog is out of scope of this ticket, but it is logical to handle the link to the job description similarly to how we would handle other table links.